packages/devtools_app/lib/src/screens/memory/panes/profile/model.dart

```diff
@@ -84,32 +115,70 @@ class AdaptedProfile {
   final GCStats totalGCStats;
 }
 
+/// Constants ar short here, as they repeat and we want to save space.
```

Comment:
r/ar/are

also, if this is a comment, use // instead of ///. /// is for dart doc that defines the class or member